pkgs/tools/misc/lavat/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+,
+}:
+let
+  version = "2.0.0";
+in
+stdenv.mkDerivation {
+  pname = "lavat";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "AngelJumbo";
+    repo = "lavat";
+    rev = "v${version}";
+    sha256 = "sha256-xDjqKhwoaqCqo7tkpcEe2MBEpVTJUOpKtu7Fi9aPOPo=";
```

Comment:
is this a normally accepted style choice? I've found inconsistent use of hash and sha256 in nixpkgs (including both other packages I maintain using sha256) so I'm wondering if this is now consensus

Reply:
Yes, we will have to use `hash` in the long run.

Reply:
ok awesome! I'll change this over, can I have a link to the issue/chat/place where this was decided so I can bring it up as a thing to change when version bumping other packages?

Reply:
Let me search for it...

Reply:
I've made the change, feel free to resolve this after putting the link here

Reply:
Cant find anything yet on this with my phone, but I'll fix this tomorrow!

Reply:
sure, no worries! both will work so it's just a matter of refinding a bit of documentation saying which is preferred in nixpkgs

Reply:
I found this: https://floxdev.com/blog/nixpkgs-contribution

But this is not the link I was looking for

Reply:
possibly https://github.com/NixOS/nixpkgs/pull/112388? it was a while back but suggets the hash attribute is better than sha256

I probably won't change existing packages based on it but I'm fine with using `hash =` for new ones

Reply:
I can't find the link where using SRI hashes are now the best practice.

Maybe @jtojnar knows it better?

Reply:
I am not aware of any docs on that. Many people just prefer `hash` because it is the new way of specifying `outputHash` and using `sha256` is redundant with SRI hashes.